### ISSUE TITLE
[BUG]: Reporting fails silently when browserName is not set in config

### ISSUE THREAD

**Issue by cthlo** [microsoft/playwright-testing-service#133]
**Describe the bug**
When [`browserName`](https://playwright.dev/docs/api/class-testoptions#test-options-browser-name) option is not set in config, test reporting will fail silently

```sh
❯ npx playwright test --project=project_1 -c ./playwright.service.config.ts

Running 3 tests using 3 workers


     1 [project_1] › some/test/test1.spec.ts:50:3 › test case 1
     ....

  3 passed (5.8s)

Uploading test results.
Test report: https://playwright.microsoft.com/workspaces/westus3_...
```

I'm guessing it's failing here: [`reporterUtils.ts#L161`](https://github.com/Azure/azure-sdk-for-js/blob/b7fc2175a043afce979b8dd6cc70b0dfa8815f25/sdk/playwrighttesting/microsoft-playwright-testing/src/utils/reporterUtils.ts#L161)

**To Reproduce**
1. Leave out [`browserName`](https://playwright.dev/docs/api/class-testoptions#test-options-browser-name) option in Playwright config
2. Opt out of cloud hosted browser in Microsoft Playwright Testing service config: `useCloudHostedBrowsers: false`
3. Include `@azure/microsoft-playwright-testing/reporter` reporter in config

**Expected behavior**
Error thrown to indicate that `browserName` is required, or upload test results without browser name, or maybe use an alternative way (if there's one) to get the browser name.

**Setup information (please complete the following information):**
 - Azure Subscription Id: N/A
 - Workspace Name: N/A
 - Service trace Id (if available):
 - Log snippet (if available):
 - Browser on which tests are executed (if applicable):
 - Time range (in PST) when the issue was experienced:
 - Playwright Version:
   - "@playwright/test": "^1.47.1"
   - "@azure/microsoft-playwright-testing": "^1.0.0-beta.3"
 - Client machine OS:
 - OS requested on Service:

**Comment by vvs11** [comment]
Hi @cthlo Thanks for reporting this. We are looking into this and will share an update soon

**Comment by abranj19** [comment]
Hi @cthlo , this issue has been fixed in the version 1.0.0-beta.4  . Please upgrade the package. Marking this as closed . Pls feel free to reopen in case you face this issue again.
